macros/invoke_core_deployer.sh

```diff
@@ -4,6 +4,7 @@ name=$1
 curl_opts=$2
 expected_data=$3
 
+NAMESPACE=${NAMESPACE:-fats}
```

Comment:
rm

NAMESPACE is part of the FATS contract. We should not define it here. The script should fail if the value is not set.